frontend/src/components/profile/my-tickets.tsx

```diff
@@ -0,0 +1,228 @@
+/** @jsxRuntime classic */
+
+/** @jsx jsx */
+import { useState, useCallback } from "react";
+import { FormattedMessage } from "react-intl";
+import { Box, Heading, jsx } from "theme-ui";
+
+import { useRouter } from "next/router";
+
+import { Alert } from "~/components/alert";
+import { Button } from "~/components/button/button";
+import { Modal } from "~/components/modal";
+import { Table } from "~/components/table";
+import { QuestionsSection } from "~/components/tickets-page/questions-section";
+import { useTranslatedMessage } from "~/helpers/use-translated-message";
+import { AttendeeTicket, useUpdateTicketMutation } from "~/types";
+
+import { ProductState } from "../tickets-page/types";
+import { Link } from "~/components/link";
+
+type Props = {
+  tickets?: AttendeeTicket[];
+};
+
+export const MyTickets: React.FC<Props> = ({ tickets }) => {
+  const code = process.env.conferenceCode;
+  const ticketHeader = useTranslatedMessage("profile.ticketFor");
+  const nameHeader = useTranslatedMessage("orderReview.attendeeName");
+  const emailHeader = useTranslatedMessage("orderReview.attendeeEmail");
+  const [currentTicket, setCurrentTicket] = useState({ id: null, show: false });
+
+  const headers = [ticketHeader, nameHeader, emailHeader, ""];
+
+  const [selectedProducts, setSelectedProducts] = useState(
```

Comment:
why `selectedProducts`? is this the list of tickets?

Reply:
`selectedProducts` is needed to `QuestionSection` to render correctly

Reply:
I feel like we failed to make the form generic

but doesn't make sense to work on it now since it is late, we will refactor the tickets page in the future anyway

Reply:
yeah, seems quite complicated... 😄